packages/datepickers/src/DatepickerRange/components/Calendar.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { HTMLProps } from 'react';
+import addMonths from 'date-fns/addMonths';
+
+import { StyledRangeCalendar } from '../../styled';
+import useDatepickerRangeContext from '../utils/useDatepickerRangeContext';
+import Month from './Month';
+
+const Calendar: React.FunctionComponent<HTMLProps<HTMLDivElement>> = props => {
+  const {
+    state,
+    dispatch,
+    locale,
+    small,
+    minValue,
+    maxValue,
+    startValue,
+    endValue,
+    onChange
+  } = useDatepickerRangeContext();
+
+  return (
+    <StyledRangeCalendar
+      data-garden-id="datepickers.range"
+      data-garden-version={PACKAGE_VERSION}
+      data-test-id="range-calendar"
```

Comment:
Do we ship these test-ids to our published packages?

Reply:
https://github.com/zendeskgarden/react-components/blob/master/babel.config.js#L26 removes all `data-test*` attributes from the `production` bundle. This should only be visible locally and in our tests.